scripts/zkai_ezkl_statement_envelope_benchmark.py

```
@@ -0,0 +1,471 @@
+
+
+def ezkl_verify(
+    proof: dict[str, Any],
+    settings_path: pathlib.Path,
+    vk_path: pathlib.Path,
+    srs_path: pathlib.Path,
+) -> None:
+    try:
+        import ezkl  # type: ignore
+    except ImportError as err:
+        raise EzklEnvelopeError("ezkl Python package is not installed") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)